backend/src/models/reports/Tab_32_rpt_PA_Registered.js

```diff
@@ -1,16 +1,132 @@
-//const dbConnection = require("@database/databaseConnection");
-//const { knex } = dbConnection();
+// libs
+const { knex } = require("@database/databaseConnection")();
+const log = require("../../facilities/logging")(module.filename);
+const _ = require("lodash");
 
-const getReport = (portfolio) => {
-  const query = "removeme"; //knex().raw();
-  return query;
-};
+// utilities
+const { formatDate, groupByProperty } = require("./helpers");
 
-module.exports = {
-  required: [],
-  getAll: async (query) => {
-    const { portfolio } = query;
-    const [report] = await Promise.all([getReport(portfolio)]);
-    return { report };
+/**
+ * Retrieves the data for the Projects Registered Report.
+ *
+ * @param   {string} date - The date to grab data for.
+ * @returns {object}      - An object containing the report, report totals, and the date.
+ * @throws  {Error}       - A generic error message with no details.
+ */
+const queries = {
+  // The columns on which to calculate totals.
+  columns: {
+    total: "report.planned_budget",
   },
+
+  // The query to get the report data.
+  report: (date) => {
+    const query = knex
+      .select({
+        portfolio_name: "po.portfolio_name",
+        "#": "p.project_number",
```

Comment:
have we used the # as a label before?

Reply:
nope, and I don't like it much.

It was in gary's query though.

![image](https://github.com/bcgov/gdx-agreements-tracker/assets/122112933/dd6c393c-cf64-4229-9231-81294d8fb362)